ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/components/video-view/video-view.component.ts

```diff
@@ -52,6 +53,7 @@ export class VideoViewComponent implements OnDestroy {
   private stopPlaying() {
     this.videoTag.pause();
     this.videoTag.currentTime = 0;
+    this.videoTag.muted = true;
```

Comment:
This will leave the video streaming....just muted ?

Reply:
yes, I've tested and it works for now.

Reply:
There may be other solutions, but this works.